cropduster/files.py

```diff
@@ -88,11 +89,10 @@ def __init__(self, path, upload_to=None, preview_w=None, preview_h=None):
             # url on other server? download it.
             self._path = self.download_image_url(path)
         else:
-            abs_path = get_media_path(path)
-            if os.path.exists(abs_path):
-                self._path = get_relative_media_url(abs_path)
+            if default_storage.exists(path):
```

Comment:
Since this is a FileField, should we be using [self.storage](https://github.com/django/django/blob/828e3b1335e2614d338a630fd5b5f88d38a6b5d2/django/db/models/fields/files.py#L19)?

Reply:
In places where you don't have easy access to the field, there is a utility function to get it from the model and field identifier:

https://github.com/theatlantic/django-cropduster/blob/38ab3877897029ff5065284c44a4ba1a70316691/cropduster/forms.py#L144-L164

Reply:
To sort-of answer my own question, I think the answer is probably 'no', since the `ImageField` on `cropduster.Image` doesn't have a configurable storage.